docs/api/content-tracing.md

```diff
@@ -122,13 +122,22 @@ all child processes to flush any pending trace data.
 Once all child processes have acknowledged the `captureMonitoringSnapshot`
 request the `callback` will be called with a file that contains the traced data.
 
-
 ### `contentTracing.getTraceBufferUsage(callback)`
 
 * `callback` Function
-  * `value` Number
-  * `percentage` Number
+  * Object
+    * `value` Number
+    * `percentage` Number
 
 Get the maximum usage across processes of trace buffer as a percentage of the
 full state. When the TraceBufferUsage value is determined the `callback` is
 called.
+
+**[Deprecated Soon](promisification.md)**
+
+### `contentTracing.getTraceBufferUsage()`
+
+Returns `Promise<Object>` - Resolves with an object containing the `value` and `percentage` of trace buffer maximum usage
```

Comment:
I would wait for the `electron.d.ts` generation to be fixed in master to be able to check if it's going to be generated properly.